supported_build_configurations.yaml

```diff
@@ -61,7 +61,7 @@ test-lookup-table:
   - nucleo
 
 test-adafruitSTEMMA:
-#  - nucleo
+  #  - nucleo
```

Comment:
tip for your coops and here: avoid changes like these. Every change in your PRs should be deliberate. please undo this change